crates/breez-sdk/breez-itest/tests/tokens.rs

```diff
@@ -0,0 +1,955 @@
+use anyhow::Result;
+use breez_sdk_itest::*;
+use breez_sdk_spark::*;
+use rand::RngCore;
+use rstest::*;
+use tempdir::TempDir;
+use tracing::info;
+
+async fn create_mint_test_token(instance: &SdkInstance) -> Result<TokenMetadata> {
+    let issuer = instance.sdk.get_token_issuer();
+    let token_metadata = issuer
+        .create_issuer_token(CreateIssuerTokenRequest {
+            name: "breez-itest token".to_string(),
+            ticker: "BIT".to_string(),
+            decimals: 2,
+            is_freezable: false,
+            max_supply: 1_000_000,
+        })
+        .await?;
+
+    issuer
+        .mint_issuer_token(MintIssuerTokenRequest { amount: 1_000_000 })
+        .await?;
+
+    info!("Minted 1,000,000 tokens");
+
+    tokio::time::sleep(std::time::Duration::from_secs(1)).await;
+    instance.sdk.sync_wallet(SyncWalletRequest {}).await?;
+    Ok(token_metadata)
+}
+
+// ---------------------
+// Fixtures
+// ---------------------
+
+/// Fixture: Alice's SDK with temporary storage
+#[fixture]
+async fn alice_sdk() -> Result<SdkInstance> {
+    let alice_dir = TempDir::new("breez-sdk-alice")?;
+    let path = alice_dir.path().to_string_lossy().to_string();
+
+    // Generate random seed for Alice
+    let mut seed = [0u8; 32];
+    rand::thread_rng().fill_bytes(&mut seed);
+
+    info!("Initializing Alice's SDK at: {} with random seed", path);
+    build_sdk_with_dir(path, seed, Some(alice_dir)).await
+}
+
+/// Fixture: Bob's SDK with temporary storage
+#[fixture]
+async fn bob_sdk() -> Result<SdkInstance> {
+    let bob_dir = TempDir::new("breez-sdk-bob")?;
+    let path = bob_dir.path().to_string_lossy().to_string();
+
+    // Generate random seed for Bob
+    let mut seed = [0u8; 32];
+    rand::thread_rng().fill_bytes(&mut seed);
+
+    info!("Initializing Bob's SDK at: {} with random seed", path);
+    build_sdk_with_dir(path, seed, Some(bob_dir)).await
+}
```

Comment:
There are reusable fixtures now for alice and bob in `crates/breez-sdk/breez-itest/src/fixtures.rs` if you want to use them

Reply:
Applied, thanks